src/ncp/ncp_openthread.cpp

```diff
@@ -209,6 +219,18 @@ otError ControllerOpenThread::ApplyFeatureFlagList(const FeatureFlagList &aFeatu
     otNat64SetEnabled(mInstance, aFeatureFlagList.enable_nat64());
 #endif
 
+#if OTBR_ENABLE_DETAILED_LOGGING_FLAG
```

Comment:
I wonder when we want to set this macro to 0?

Reply:
Like to check, does it make sense to remove OTBR_ENABLE_DETAILED_LOGGING_FLAG if we always want to enable it, and it has small memory/cpu usage?

Reply:
I guess this feature is only wanted when feature flags are supported, maybe I can remove OTBR_ENABLE_DETAILED_LOGGING_FLAG and use OTBR_ENABLE_FEATURE_FLAGS instead?

Reply:
I think you don't need to add guard for `OTBR_ENABLE_FEATURE_FLAGS` since the whole function has already been guarded by it.

Reply:
Will remove the flag

Reply:
Suggest to still use OTBR_ENABLE_DETAILED_LOGGING_FLAG since it is more readable,  OTBR_ENABLE_FEATURE_FLAGS is  not a good name for your new feature. Or remove all OTBR_ENABLE_DETAILED_LOGGING_FLAG if it is not needed.

Reply:
Flag removed.